.github/workflows/test-changed-auth.yml

```diff
@@ -23,7 +23,7 @@ env:
   # the behavior to use the new URLs.
   CHROMEDRIVER_CDNURL: https://googlechromelabs.github.io/
   CHROMEDRIVER_CDNBINARIESURL: https://storage.googleapis.com/chrome-for-testing-public
-  CHROME_VALIDATED_VERSION: linux-120.0.6099.71
+  CHROME_VALIDATED_VERSION: linux-137.0.7151.119
```

Comment:
I think we should also update https://github.com/firebase/firebase-js-sdk/blob/main/.github/workflows/test-all.yml#L26 to have the same version.